euth/phases/models.py

```diff
@@ -17,8 +17,8 @@ def active_phases(self):
 
 
 class Phase(models.Model):
-    name = models.CharField(max_length=512)
-    description = models.TextField()
+    name = models.CharField(max_length=80)
```

Comment:
Those changes require data migrations, otherwise they might fail if in the database there are already longer char fields.